sycl/doc/extensions/experimental/sycl_ext_oneapi_memory_export.asciidoc

```diff
@@ -0,0 +1,388 @@
+|1   | Initial draft of the proposal
+|======================
+
+=== Querying device support
+
+We provide the following device aspect to query for support of exporting memory.
+
+[frame="none",options="header"]
+|======================
+|Device descriptor |Description
+|`aspect::ext_oneapi_exportable_device_mem` | Indicates if the device supports
+the allocation of exportable linear memory and exporting that memory to an
+interoperable handle.
+|======================
+
+=== External Memory Resource Handle Types [[external_mem_res_handles]]
+
+This extension provides an enum `external_mem_handle_type` that defines several
+external memory resource handle types that can be used as interoperable
+handles to import SYCL allocated memory into external APIs.
+
+[_Note:_ Not all of the handle types defined in this enum may be supported for
+exporting memory by the implementation. Currently, the {dpcpp} implementation
+only supports exporting memory with the `opaque_fd` and `win32_nt_handle` handle
+types. This enum is shared with the memory import functionality defined in the
+https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc[sycl_ext_oneapi_bindless_images]
+extension, where more handle types may be supported for importing memory into
+SYCL.
+_{endnote}_]
+
+```c++
+namespace sycl::ext::oneapi::experimental {
+
+// External memory resource handle types.
+enum class external_mem_handle_type {
+  opaque_fd = 0,
+  win32_nt_handle = 1,
+  win32_nt_dx12_resource = 2,
+  dma_buf = 3,
+  win32_nt_dx11_resource = 4,
+};
+
+template <external_mem_handle_type ExternalMemHandleType>
+struct exported_mem;
+
+template <>
+struct exported_mem<external_mem_handle_type::opaque_fd> {
+  using type = int;
+};
+
+template <>
+struct exported_mem<external_mem_handle_type::win32_nt_handle> {
+  using type = void *;
+};
+
+template <external_mem_handle_type ExternalMemHandleType>
+using exported_mem_t = typename exported_mem<ExternalMemHandleType>::type;
+
+}
+```
+
+The `external_mem_handle_type` enum class defines the types of external memory
+resource handles that can be exported by this extension. The `opaque_fd` and
+`win32_nt_handle` values are used during allocation of exportable memory to
+indicate the type of handle that will later be returned by the
+`export_device_mem_handle` function.
+
+The `opaque_fd` handle type corresponds to a POSIX file descriptor, which is
+represented by an `int`.
+
+The `win32_nt_handle` handle type corresponds to a Windows NT handle, which is
+represented by a `void *`.
+
+The `exported_mem` template struct is specialized for each of the
+`external_mem_handle_type` values. Each specialization defines a `type` alias
+that represents the type of the exported memory handle for that specific handle.
+
+The `exported_mem_t` type alias allows the user to easily obtain the type of
+the exported memory handle based on the `external_mem_handle_type` template
+parameter.
```

Comment:
And remove the descriptions of them too.

Reply:
Removed.